alfresco-remote-testrunner/build.gradle

```
@@ -35,6 +35,7 @@ task shadowJar(type: ShadowJar) {
     classifier = "shadowjar"
     from(sourceSets.main.output)
     configurations = [project.configurations.shadowed]
+    relocate 'org.apache.http.client.fluent', 'shadow.relocated.org.apache.http.client.fluent'
```

```suggestion
    relocate 'org.apache.http.client', 'eu.xenit.testing.integrationtesting.internal.shadow.org.apache.http.client'
```

- We don't only need to shadow the dependent library, but also the dependencies.
- And we want to move it to an internal namespace of **our** library, not to some generic namespace (to avoid conflicts with other libraries that also shadow dependencies)